product_margin_classification/__manifest__.py

```diff
@@ -5,11 +5,26 @@
 
 {
     'name': 'Product Margin Classification',
-    'version': '8.0.2.1.0',
+    'version': '10.0.1.0.0',
     'category': 'Account',
     'author': 'GRAP,Odoo Community Association (OCA)',
-    'website': 'http://www.grap.coop',
+    'website': 'https://github.com/OCA/sale-workflow',
     'license': 'AGPL-3',
+    'description': """
```

Comment:
this 'description' is useless, as there is the readme folder. when merging your PR, the readme.rst file will be generated by bot.